src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import { Comment } from '../../types/comment';
+import { CommentInfo } from '../CommentInfo';
+
+type CommentsType = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<CommentsType> = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      <div className="CommentInfo">
+        {comments.map(comment => (
+          <CommentInfo key={comment.id} comment={comment} />
```

Comment:
```suggestion
  <div className="CommentList">
    {comments.map(comment => (
      <CommentInfo key={comment.id} comment={comment} />
```
React fragment is actually redundant. Also, move CommentInfo block inside CommentInfo component